src/Pages/DashboardPage/DashboardPage.jsx

```diff
@@ -1,33 +1,69 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuthContext } from "../../Context/Auth.context";
+import { useDocTitle } from "../../Hook/useTitle";
+import { LandingPage } from "../LandingPage/LandingPage";
 import "./DashboardPage.css";
 
 export const DashboardPage = () => {
+  useDocTitle("Dashboard | StormKeep")
+  const encodedToken = localStorage.getItem("StormKeepToken");
+  const [route, setRoute] = useState("profile");
+  const { userState, signout } = useAuthContext();
+
   return (
     <div className="showcase-container">
       <div className="dashboard">
         <h1 className="dashboard-header">Account</h1>
-        <div className="dasboard-container">
+        <div className="dashboard-container">
           <div className="dashboard-sections">
             <ul>
-              <li>Profile</li>
-              <li>Settings</li>
+              <li
+                onClick={() => setRoute("profile")}
+                className={route === "profile" ? "active" : null}
+              >
+                Profile
+              </li>
+              <li
+                onClick={() => setRoute("settings")}
+                className={route === "settings" ? "active" : null}
+              >
+                Settings
+              </li>
             </ul>
           </div>
           <div className="dashboard-section-details">
             <div className="profile-details">
-              <h2>Profile Details</h2>
-              <div className="txt-container">
-                <div className="text-row">
-                  <span>Name:</span>
-                  <span>12313</span>
-                </div>
-                <div className="text-row">
-                  <span>Email:</span>
-                  <span>123123</span>
-                </div>
-              </div>
-
-              <h2>Account Settings</h2>
-              <button>Log Out</button>
+              {route === "profile" ? (
+                <>
+                  <h2>Profile Details</h2>
+                  <div className="txt-container">
+                    <div className="text-row">
+                      <span>Name:</span>
+                      <span>{userState.firstName}</span>
+                    </div>
+                    <div className="text-row">
+                      <span>Email:</span>
+                      <span>{userState.email}</span>
+                    </div>
+                  </div>
+                </>
+              ) : null}
+              {route === "settings" ? (
+                <>
+                  <h2>Account Settings</h2>
+                  <Link to="/" element={<LandingPage />}>
+                    {encodedToken && (
+                      <button
+                        onClick={() => signout()}
```

Comment:
suggestion - as `signout` does not accept any arguments, you can just write `onClick = {signout}`

Reply:
yes thanks for pointing it out. I actually forgot to change it back!